plugins/outputs/appdynamics/appdynamics.go

```diff
@@ -0,0 +1,127 @@
+package influxdb
+
+import (
+	"encoding/json"
```

Comment:
Also, each group of imports should be sorted, for better readability (my editor does it automatically, not sure what you're using).